player/stream-test/js/script.js

```diff
@@ -128,44 +239,324 @@ function check404(url, callbackFunction) {
   xhr.open('GET', url, true);
   xhr.onreadystatechange = function () {
     if (xhr.readyState === XMLHttpRequest.DONE) {
-        xhr.onreadystatechange = null;
```

Comment:
[`fetch`](https://caniuse.com/#feat=fetch) is also widely supported now, so that can be used too.

Reply:
`fetch` is not supported by IE that is why this polyfill was created :)